src/widgets/VaccineChart.js

```diff
@@ -155,40 +265,77 @@ export class VaccineChart extends React.Component {
   }
 }
 
+const chartSize = {
+  width: 1,
+  height: 1.025,
+};
+
 const chartStyles = {
   display: 'flex',
   flexDirection: 'row',
   justifyContent: 'center',
 };
 
+const axisStyles = {
+  fontSize: 15,
+  fontFamily: APP_FONT_FAMILY,
+  fill: '#909192',
```

Comment:
this color is 'GREY' can import it alongside  APP_FONT_FAMIL etc.

Reply:
Fixed by 4a915d3.